front/lib/labs/transcripts/utils/helpers.ts

```diff
@@ -48,3 +48,12 @@ export async function getGoogleAuthFromUserTranscriptsConfiguration(
     transcriptsConfiguration.connectionId
   );
 }
+
+export async function getAccessTokenFromNango(
```

Comment:
Are you sure you don't need to cache version like we have here:
https://github.com/dust-tt/dust/blob/3687267c63af97c1777bc463440685758eef7c78/connectors/src/lib/nango_helpers.ts#L9-L29